lib/Runtime/Language/JavascriptExceptionOperators.h

```diff
@@ -47,6 +47,7 @@ namespace Js
         {
           private:
             void * m_prevTryCatchFrameAddr;
+            bool m_prevDoneBailOutEhRegion;
```

Comment:
Nit: is this used in the current version of the changes?

Reply:
No, removed it. Thanks for catching this.